libs/vue-route-guard/src/lib/helpers.ts

```diff
@@ -38,7 +38,7 @@ export function getVueVersion(app: typeof Vue): number {
 export function isStorageAvailable(storage: StorageType): boolean {
   const storageValue = window[storage];
   if (
-    !!storageValue ||
+    !storageValue ||
```

Comment:
could you please add a test for that?

Reply:
There's a ticket to add tests to the package: https://github.com/thisdot/open-source/issues/125